### DISCUSSION TITLE
Supported version of Python

### DISCUSSION THREAD

**Discussion by kookma** [DanWBR/dwsim#601] [Q&A]
How can I find what version of python is supported by the DWSIM latest release?
I am on Windows 11, and have python 3.11 and 3.12 on my system, but none of them works and I cannot find which version is supported.

I set the path to python 3.x dll directory (windows only) in:
General Settings --> Python -->Python.NET Interpreter settings

**Comment by DanWBR** [comment]
@kookma 3.6 to 3.9, because of a limitation in the Reaktoro Gibbs Reactor model. You can use a portable distro like WinPython

**Comment by kookma** [comment]
Thank you @DanWBR 
Is it possible to show this somewhere? For example where user sets the path.

**Reply by DanWBR** [reply]
@kookma you should see an error message with this information once the python environment is activated

**Comment by DanWBR** [comment]
@kookma I added this

![Captura de Tela 2024-03-08 às 12 32 57](https://github.com/DanWBR/dwsim/assets/2669419/7e645f1d-4707-4d3b-915c-4b213e345ed8)

**Comment by kookma** [comment]
Much appreciated.
